Source/Plugins/Core/com.equella.core/js/__tests__/tsrc/search/SearchPage.test.tsx

```diff
@@ -320,6 +317,8 @@ describe("Refine search by status", () => {
   const selectStatus = (container: Element, status: string) =>
     fireEvent.click(getByText(getStatusSelector(container), status));
 
+  const liveStatuses = SearchModule.liveStatuses;
+  const nonLiveStatuses = SearchModule.nonLiveStatuses;
```

Comment:
Destructuring can deduplicate this a little

```suggestion
  const { liveStatuses, nonLiveStatuses } = SearchModule;
```